polkadot/xcm/pallet-xcm-benchmarks/src/fungible/benchmarking.rs

```diff
@@ -79,7 +82,7 @@ benchmarks_instance_pallet! {
 	}: {
 		executor.bench_process(xcm)?;
 	} verify {
-		assert!(T::TransactAsset::balance(&sender_account).is_zero());
+		assert!(T::TransactAsset::balance(&sender_account) < sender_account_balance_before);
```

Comment:
Is this a tight enough bound on this test? It's just checking that it decreases by any amount, whereas do we not expect it to change by some predictable amount?

Reply:
I agree it should be tighter. Will look into it on another PR 😄